examples/with-zones/blog/next.config.js

```diff
@@ -1,6 +1,14 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   basePath: "/blog",
+  experimental: {
+    // Next.js will automatically prefix `basePath` to client side links which
+    // is useful when all links are relative to the `basePath` of this
+    // application. This option opts out of that behavior, which can be useful
+    // if you want to link outside of your zone, such as linking to
+    // "/" from "/blog" (the `basePath` for this application).
+    manualClientBasePath: true,
```

Comment:
Should we document this option?

Reply:
I will follow up with the team after this to see if we want to move this out of experimental or pursue other options, like adding a parameter to the `Link` element.